components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.html

```diff
@@ -41,15 +41,18 @@
     <ul>
       <li class="service-item" *ngFor="let service of services$ | async">
         <div class="header">
-          <div class ="fqdn-line"><chef-icon>storage</chef-icon> {{ service.fqdn || "- -"}}</div>
-          <div class ="supervisor-line"><b>Supervisor:</b> {{ service.supervisor_id }}</div>
-          <div class ="site-line"><b>Site:</b> {{ service.site  || "- -"}}</div>
+          <div class ="fqdn-line"><chef-icon>storage</chef-icon> {{ service.fqdn || "--"}}</div>
+          <div class ="supervisor-line"><b>Supervisor:</b> {{ service.supervisor_id || "--"}}</div>
+          <div class ="site-line"><b>Site:</b> {{ service.site  || "--"}}</div>
         </div>
         <div class="body">
-          <div class ="release-line"><chef-icon>grain</chef-icon>{{ service.release }}</div>
+          <div class ="release-line"><chef-icon>grain</chef-icon>{{ service.release || "--"}}</div>
```

Comment:
All these `--` (no space) updates are needed since the mockups say that is the right way to display empty data fields: https://chef.invisionapp.com/share/PZQMKLHYC8B#/screens/361825016_data_empty